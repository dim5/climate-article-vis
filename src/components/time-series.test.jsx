import { render } from '@testing-library/react';
import './__mocks__/resize-observer.mock.js';

import TimeSeries from './time-series';

const entities = ['AUSTRALIA', 'CALIFORNIA'];

it('should render', () => {
  const { container } = render(<TimeSeries entities={entities} />);
  expect(container.innerHTML).toMatchSnapshot();
});
