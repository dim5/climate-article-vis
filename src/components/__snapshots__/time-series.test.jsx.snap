// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `"<div class=\\"time-series__TimeSeriesContainer-sc-1efg098-1 dKPNlD\\"><div class=\\"line-chart__LineChartContainer-sc-12ede8g-0 bCToO time-series___StyledLineChart-sc-1efg098-2 ccdaRm\\"><canvas style=\\"display: block; box-sizing: border-box; width: 0px; height: 0px;\\" height=\\"0\\" width=\\"0\\"></canvas></div><div class=\\"ant-radio-group ant-radio-group-solid time-series__StyledMenu-sc-1efg098-0 jLbcMb\\"><label class=\\"ant-radio-button-wrapper\\"><span class=\\"ant-radio-button\\"><input type=\\"radio\\" class=\\"ant-radio-button-input\\" value=\\"day\\"><span class=\\"ant-radio-button-inner\\"></span></span><span>Daily</span></label><label class=\\"ant-radio-button-wrapper ant-radio-button-wrapper-checked\\"><span class=\\"ant-radio-button ant-radio-button-checked\\"><input type=\\"radio\\" class=\\"ant-radio-button-input\\" value=\\"week\\" checked=\\"\\"><span class=\\"ant-radio-button-inner\\"></span></span><span>Weekly</span></label><label class=\\"ant-radio-button-wrapper\\"><span class=\\"ant-radio-button\\"><input type=\\"radio\\" class=\\"ant-radio-button-input\\" value=\\"month\\"><span class=\\"ant-radio-button-inner\\"></span></span><span>Monthly</span></label></div></div>"`;
